package.json

```diff
@@ -41,7 +42,7 @@
     "mkdirp": "0.5.1",
     "mv": "2.1.1",
     "pkginfo": "0.4.1",
-    "request": "2.88.0",
+    "request": "2.87.0",
```

Comment:
why we downgrade this package?

Reply:
There is a lot of issues reported with that version, it makes fails a bunch of test for some reason. 

https://github.com/request/request/issues/3072

Reply:
https://github.com/request/request/issues/3085

Reply:
👍